_security-plugin/access-control/api.md

```diff
@@ -1179,110 +1179,147 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
-
 ---
 
-## Certificates
+## Distinguished names
 
-### Get certificates
-Introduced 1.0
-{: .label .label-purple }
+These REST APIs let a super admin allow list distinguished names to enable communication between clusters and/or nodes.
```

Comment:
So if we want to use the noun form, absolutely, "allow list" seems fine. But you've verbed it here, which makes me think it either needs to be "allowlist" or rephrased to something like "let a super admin list the distinguished names that are allowed to enable communication between clusters and/or nodes."

I get that we already use "allow list" as a verb in a couple places. Check with editorial and make consistent everywhere?

Reply:
Turns out we just _don't_ use "allow list" as a verb, so I've changed all "allow list" references to only be nouns.